sw/device/silicon_creator/rom_ext/prodc/BUILD

```diff
@@ -0,0 +1,137 @@
+# in the manifest definition).  You can manually create such a bitstream with:
+#
+# bazel build //hw/bitstream/universal:splice --//hw/bitstream/universal:env=//hw/top_earlgrey:fpga_cw310_sival
+opentitan_binary(
+    name = "rom_ext_fake_prod_signed_slot_a",
+    exec_env = [
+        "//hw/top_earlgrey:silicon_creator",
+        "//hw/top_earlgrey:fpga_cw310",
+        "//hw/top_earlgrey:sim_dv_base",
+        "//hw/top_earlgrey:sim_verilator_base",
+    ],
+    linker_script = "//sw/device/silicon_creator/rom_ext:ld_slot_a",
+    linkopts = LINK_ORDER,
+    manifest = ":manifest_prodc",
+    rsa_key = {"//sw/device/silicon_creator/rom/keys/fake/rsa:prod_private_key_0": "prod_key_0"},
+    deps = [
+        "//sw/device/lib/crt",
+        "//sw/device/silicon_creator/lib:manifest_def",
+        "//sw/device/silicon_creator/rom_ext",
+        "//sw/device/silicon_creator/rom_ext/keys/fake",
+    ],
+)
+
+opentitan_binary(
+    name = "rom_ext_fake_prod_signed_slot_b",
+    exec_env = [
+        "//hw/top_earlgrey:silicon_creator",
+        "//hw/top_earlgrey:fpga_cw310",
+        "//hw/top_earlgrey:sim_dv_base",
+        "//hw/top_earlgrey:sim_verilator_base",
+    ],
+    linker_script = "//sw/device/silicon_creator/rom_ext:ld_slot_b",
+    linkopts = LINK_ORDER,
+    manifest = ":manifest_prodc",
+    rsa_key = {"//sw/device/silicon_creator/rom/keys/fake/rsa:prod_private_key_0": "prod_key_0"},
+    deps = [
+        "//sw/device/lib/crt",
+        "//sw/device/silicon_creator/lib:manifest_def",
+        "//sw/device/silicon_creator/rom_ext",
+        "//sw/device/silicon_creator/rom_ext/keys/fake",
+    ],
+)
+
+opentitan_binary(
+    name = "rom_ext_real_prod_signed_slot_a",
+    exec_env = [
+        "//hw/top_earlgrey:silicon_creator",
+        "//hw/top_earlgrey:fpga_cw310",
+    ],
+    linker_script = "//sw/device/silicon_creator/rom_ext:ld_slot_a",
+    linkopts = LINK_ORDER,
+    deps = [
+        "//sw/device/lib/crt",
+        "//sw/device/silicon_creator/lib:manifest_def",
+        "//sw/device/silicon_creator/rom_ext",
+        "//sw/device/silicon_creator/rom_ext/prodc/keys",
+    ],
+)
+
+opentitan_binary(
+    name = "rom_ext_real_prod_signed_slot_b",
+    exec_env = [
+        "//hw/top_earlgrey:silicon_creator",
+        "//hw/top_earlgrey:fpga_cw310",
+    ],
+    linker_script = "//sw/device/silicon_creator/rom_ext:ld_slot_b",
+    linkopts = LINK_ORDER,
+    deps = [
+        "//sw/device/lib/crt",
+        "//sw/device/silicon_creator/lib:manifest_def",
+        "//sw/device/silicon_creator/rom_ext",
+        "//sw/device/silicon_creator/rom_ext/prodc/keys",
+    ],
+)
+
+offline_presigning_artifacts(
+    name = "presigning",
+    testonly = True,
+    srcs = [
+        ":rom_ext_real_prod_signed_slot_a",
+        ":rom_ext_real_prod_signed_slot_b",
+    ],
+    manifest = ":manifest_prodc",
+    rsa_key = {
+        "//sw/device/silicon_creator/rom/keys/real/rsa:earlgrey_a0_prod_0": "ealrgrey_a0_prod_0",
```

Comment:
What do you think about using `prod_2` for this configuration?

Reply:
Done: set to `earlgrey_a0_prod_2`.